gnosis/src/main/scala/edu/uci/ics/cloudberry/gnosis/IGeoIndex.scala

```diff
@@ -41,4 +41,32 @@ class USGeoJSONIndex() extends IGeoIndex {
   override def search(envelope: Envelope): Seq[IUSGeoJSONEntity] = {
     index.query(envelope).asScala.map(item => entities(item.asInstanceOf[Int]))
   }
+}
+
+class NYGeoJSONIndex() extends IGeoIndex {
```

Comment:
what's the difference of this `NYGeoJsonIndex`? It seems should be an instantiated object of `USGeoJsonIndex` other than a separate class.